delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/stream/CatsStreamConverter.scala

```diff
@@ -0,0 +1,89 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.stream
+
+import akka.NotUsed
+import akka.stream._
+import akka.stream.scaladsl.{Sink => AkkaSink, Source => AkkaSource, _}
+import cats.effect._
+import cats.syntax.all._
+import fs2._
+
+/**
+  * Converts a fs2 stream to an Akka source Original code from the streamz library from Martin Krasser (published under
+  * Apache License 2.0):
+  * https://github.com/krasserm/streamz/blob/master/streamz-converter/src/main/scala/streamz/converter/Converter.scala
+  */
+object CatsStreamConverter {
```

Comment:
Yet another copy of a class existing for Monix for Cats Effects.

Reply:
Why not migrate it? Seems like it would have only a small impact outside of archives?

Reply:
It is used by SSEs and composite views too for the moment